# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/zoom/browser_zoom_commands.js
# Commit: febdcf7986a3
# Full Hash: febdcf7986a37651188476cd8253d20e9cea443c
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 00d1c650d88f -r febdcf7986a3 browser/base/content/test/zoom/browser_zoom_commands.js
--- a/browser/base/content/test/zoom/browser_zoom_commands.js	Wed Jun 22 19:26:58 2022 +0000
+++ b/browser/base/content/test/zoom/browser_zoom_commands.js	Wed Jun 22 19:40:17 2022 +0000
@@ -79,6 +79,7 @@
     // and the other without.
     for (let textZoom of [true, false]) {
       info(`Running variation with textZoom set to ${textZoom}`);
+
       await SpecialPowers.pushPrefEnv({
         set: [["browser.zoom.full", !textZoom]],
       });